c/tskit/tables.c

```diff
@@ -4960,66 +4982,63 @@ tsk_table_sorter_sort_mutations_canonical(tsk_table_sorter_t *self)
 }
 
 static int
-tsk_table_sorter_sort_individuals(tsk_table_sorter_t *self)
+tsk_individual_table_count_descendants(
```

Comment:
Hmm, it's not just counting descendents though. How about ``tsk_individual_table_topological_sort``? This returns a topological sorting of the individuals and optionally a count of their descendents.